tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -0,0 +1,164 @@
+
+
+![ push to repository triggers cloud build with packer which builds machine image](packer-tutorial.png)
+
+[![button](http://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/community&page=editor&tutorial=tutorials/create-cloud-build-image-factory-using-packer/index.md)
+
+
+## Task 0 (OPTIONAL) Create a project with a billing account attached 
+This task will help you setup a new GCP project in which to run your packer build factory. 
+**(you can also use an existing project and skip to the next step)**
+```sh
+PROJECT=[NEW PROJECT NAME]
+ORG=[YOUR ORGANIZATION NAME]
+BILLING_ACCOUNT=[YOUR_BILLING_ACCOUNT_NAME]
+ZONE=[COMPUTE ZONE YOU WANT TO USE]
+ACCOUNT=[GOOGLE ACCOUNT YOU WANT TO USE] or $(gcloud config get-value account)
+```
+```sh
+gcloud projects create "$PROJECT" --organization=$(gcloud organizations list --format="value(name)" --filter="(displayName='$ORG')")
+gcloud beta billing projects link $PROJECT --billing-account=$(gcloud alpha billing accounts list --format='value(name)' --filter="(displayName='$BILLING_ACCOUNT')")
+gcloud config configurations create --activate $PROJECT
+gcloud config set project $PROJECT
+gcloud config set compute/zone $ZONE
+gcloud config set account $ACCOUNT
+```
+
+## Task 1  Set the project variable (Skip this step if you created a new project above)
+Ensure you are working with the project you want to use in gcloud.  
+For more information on configuraitons see [configurations](https://cloud.google.com/sdk/gcloud/reference/config/configurations/) 
+```sh
+gcloud config configurations activate $MY_CONFIGURATION #The configuration for the project you want to use
+PROJECT=$(gcloud config get-value project)
+```
+
+## Task 2  Enable the required services
+```sh
+gcloud services enable sourcerepo.googleapis.com
+gcloud services enable cloudapis.googleapis.com 
+gcloud services enable compute.googleapis.com
+gcloud services enable servicemanagement.googleapis.com 
+gcloud services enable storage-api.googleapis.com  
+gcloud services enable cloudbuild.googleapis.com
+```
+
+## Task 3 Give the Cloud Build user Project Editor permissions 
+```sh
+CLOUD_BUILD_ACCOUNT=$(gcloud projects get-iam-policy $PROJECT --filter="(bindings.role:roles/cloudbuild)"  --flatten="bindings[].members" --format="value(bindings.members[])")
+
+gcloud projects add-iam-policy-binding $PROJECT \
+  --member $CLOUD_BUILD_ACCOUNT \
+  --role roles/editor   
+```
+
+## Task 4 Create the Cloud Source Repository for your image creator 
+```sh
+gcloud source repos create helloworld-image-factory
+```
+
+## Task 5 Create the build triggers for the image creator source repository 
+Go to to the [build triggers page](https://console.cloud.google.com/cloud-build/triggers) and create a trigger.  
```

Comment:
Something like:

You can create a trigger on the build triggers page of the GCP Console by following these steps:

It's nice to orient the reader to where we're linking them. When I see build triggers page without further context, I'm not really sure where I'm going (*I* assume the console, but that's only because I'm steeped in GCP every day).